src/dataProviders/csvProvider.ts

```diff
@@ -0,0 +1,82 @@
+import csv from 'csvtojson';
+import {Cluster, Itur, Coordinate, Rule, RuleValue, Constitution} from '../types';
+
+const readCsvFile = async (filePath: string): Promise<any[]> => {
+  const data: any[] = await csv().fromFile(filePath);
+
+  return data;
+};
+
+export const readClusters = async (): Promise<Cluster[]> => {
+  const clustersCsvData = await readCsvFile('./data/Tvirim.csv');
+
+  const clusters = clustersCsvData.map((clusterData) => createCluster(clusterData));
```

Comment:
We can a shorthand syntax (also for Iturim and Rules):
`clustersCsvData.map(createCluster);`

Reply:
Right !
Done